### ISSUE TITLE
The new set_personalization for template is not sending emails

### ISSUE THREAD

**Issue by prateekshac1** [mailersend/mailersend-python#146]
After last night's release, set_simple_personalization started to throw an error, for template based emails. I looked up the documentation here and replaced it with set_personalization and that removed the error, and the code seems to be running successfully. But, when when i try sending a mail to myself, it's not going. I check in mailersend activity tab as well, and i see the mail not sent. Am i missing something?

**Comment by igorhrcek** [comment]
Are you getting any kind of error?

**Comment by prateekshac1** [comment]
Nope.

**Comment by prateekshac1** [comment]
Any update on this? All our production emails are stalled as a result of this. Please help resolve asap.

**Comment by igorhrcek** [comment]
@prateekshac1 I would suggest that you revert to a previous version of the library first. 

I've tested the implementation (example from the Readme) and it works as expected, haven't received any errors and email was sent regularly. I would suggest that you try using the example provided in Readme.

**Comment by prateekshac1** [comment]
yes, that solves the prod issue. i ran into trouble downgrading yesterday. But it's correctly restored now. I am using the example as in your documentation as well. Let me try a few more things. Does the error message above clue you into anything specific i should be looking for?
